ios/ConverseNotificationExtension/Xmtp/Client.swift

```diff
@@ -107,11 +107,10 @@ func putGroupInviteRequest(apiURI: String?, account: String, xmtpClient: Client,
     do {
       
       let joinRequestUri = "\(apiURI)/api/groupJoinRequest/\(joinRequestId)"
-      let privateKey = try PrivateKey(xmtpClient.keys.identityKey)
       guard let digest = "XMTP_IDENTITY".data(using: .utf8) else {
         return
       }
-      let signature = try await privateKey.sign(digest).serializedData().base64EncodedString()
+      let signature = try await xmtpClient.keys.identityKey.sign(Util.keccak256(digest)).serializedData().base64EncodedString()
```

Comment:
Looking at the [JS implementation](https://github.com/ephemeraHQ/converse-app/blob/ededda0108bcd63d7803f942c0cac7df757b666e/utils/xmtpRN/api.ts#L24) (which is what this needs to match if we want it to work), it doesn't seem like we hash the text before signing.